libs/template/helpers.go

```diff
@@ -17,4 +18,8 @@ var helperFuncs = template.FuncMap{
 	"fail": func(format string, args ...any) (any, error) {
 		return nil, ErrFail{fmt.Sprintf(format, args...)}
 	},
+	// Alias for https://pkg.go.dev/net/url#Parse. Allows usage of all methods of url.URL
+	"urlParse": func(rawUrl string) (*url.URL, error) {
```

Comment:
Should we call this just `url`? Parsing is implied, IMO.

Reply:
done